jenkins/deduper.py

```diff
@@ -41,8 +41,8 @@ def _aborted_description(current_build_id, pr):
 
         :Returns: A description (string)
         """
-        return ("[PR #{}] Build automatically aborted because"
-                " there is a newer build for the same PR. See build"
+        return ("[PR #{}] You have a newer build running for this PR."
+                " Go see its results instead! Build"
```

Comment:
Should we make the Build #{} a link?  I think the href could be obtained with `build_url = append_url(self.job.job_url, str(current_build_id))`.  (with `from helpers import append_url`)